flipperdevices/flipperzero-firmware
# [FL-3385] New RTC flags in device info

# What's new

New RTC flags in device info

- FuriHalRtcFlagDebug
- FuriHalRtcFlagLock
- FuriHalRtcFlagHandOrient
- FuriHalRtcFlagLegacySleep
- FuriHalRtcFlagStealthMode
- FuriHalRtcBootMode
- FuriHalRtcHeapTrackMode
- FuriHalRtcLocaleTimeFormat
- FuriHalRtcLocaleDateFormat
- FuriHalRtcLocaleUnits

# Verification 

- Run PR
- Go to CLI Flipper
- Choose device info(!) and see new fields 

# Checklist (For Reviewer)
